network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/article_image_block.html

```diff
@@ -1,23 +1,19 @@
 {% load wagtailimages_tags wagtailcore_tags remove_richtext_wrapping %}
 
-{% comment %} We need to break out of the main container and re-open the container at the end {% endcomment %}
-{% if self.wide_image %}
-  </div>
-{% endif %}
-
-<div class="article-image-block {% if self.wide_image %}container{% endif %}">
-  {% comment %}Use a 2x resolution image{% endcomment %}
-  {% image self.image width-1840 as img %}
-  <figure class="my-default">
-    <img src="{{ img.url }}" alt="{% if self.alt_text %}{{ self.alt_text }}{% else %}{{ img.alt }}{% endif %}" class="w-100" />
-    {% if self.caption %}
+<div class="container {{block.block_type}}-block {% if value.wide_image %} wide w-100 mw-100 px-0 mx-0{% endif %}">
+  <div class="row justify-content-center no-gutters">
+    <div class="article-image-block {% if value.wide_image %} wide {% else %} streamfield-content {% endif %} ">
+      {% comment %}Use a 2x resolution image{% endcomment %}
```

Comment:
let's remove this comment, it's not really adding anything here.

Reply:
Removed!